### PR TITLE
Update DASH SAI API comments

### PR DESCRIPTION
**Pull request by r12f**
This PR is intended to update the current comments of DASH SAI APIs.

Previously, all DASH SAI API comments are generated from P4 code using very simple rule, such as all words to upper case to pass the SAI checks. Now, we switched to python for better comment generation, which makes the comment more meaningful and readable.

### CODE REVIEW

**Review by Pterosaur — APPROVED** [review]

**Comment by Pterosaur** [comment]
Hi @prsunny , could you please help review and merge this PR?

**Comment by mukeshmv** [comment]
lgtm. This brings these SAI APIs in sync with the DASH generated APIs

**Review by marian-pritsak — APPROVED** [review]

File: inc/saiobject.h [hunk 1]
```diff
@@ -37,6 +37,9 @@
 #include <saisrv6.h>
 
 /* new experimental object type includes */
+#include <saiexperimentaldashha.h>
```
Comment (kcudnik) [inline inc/saiobject.h:40]: This is not needed here, no changes are made to this file so no object includes are required

Comment (r12f) [reply inc/saiobject.h]: Thanks Kamil. I have removed these lines now.

**Pull request merged**